ssntp/server.go

```diff
@@ -181,8 +181,23 @@ func handleSSNTPClient(server *Server, conn net.Conn) {
 		switch frame.Type {
 		case COMMAND:
 			if (Command)(frame.Operand) == CONFIGURE && session.destRole.IsController() {
-				/* TODO Send the CONFIGURE payload to the config package */
+				// setup new configuration in SSNTP server
 				server.configuration.setConfiguration(frame.Payload)
+
+				// save new configuration to be persistent
+				pyld, err := configuration.Payload(frame.Payload)
+				if err != nil {
+					server.log.Errorf("Unable to retrieve configuration payload: %v\n", err)
```

Comment:
Yes, sorry.  I misread the code.  Payload returns a structure and not  pointer to a structure so there will be no panic.  Does it make sense to send an invalid configuration package to controller though?